Automattic/woocommerce-payments
# Increase unit test coverage for WooPay direct checkout flow

Fixes 2554-gh-Automattic/woopay

#### Changes proposed in this Pull Request

This PR aims to increase the PHP and JavaScript unit test coverage for the direct to WooPay checkout flow.

Please refer to 2595-gh-Automattic/woopay for a description of the tests added for each individual issue in the project's epic.

#### Testing instructions

- Run `npm run test:js` and ensure all JavaScript tests pass.
- Run `npm run test:php` and ensure all PHP tests pass.

-------------------


**Post merge**
